### PR TITLE
hystrix: 1.3.20 → 1.5.18

### PR DESCRIPTION
**Pull request by magsel**

### CODE REVIEW

File: src/test/java/com/signicat/hystrix/servlet/TimeoutAwareHttpServletRequestTest.java [hunk 1]
```diff
@@ -9,12 +9,11 @@
 
 import org.junit.Test;
 
+import javax.servlet.ServletException;
```
Comment (einarmr) [inline src/test/java/com/signicat/hystrix/servlet/TimeoutAwareHttpServletRequestTest.java]: This is noise based on your personal IDE prefs. Please revert this file.

File: src/test/java/com/signicat/hystrix/servlet/MockHttpServletResponse.java [hunk 2]
```diff
@@ -7,16 +7,15 @@
  */
 package com.signicat.hystrix.servlet;
 
+import javax.servlet.ServletOutputStream;
```
Comment (einarmr) [inline src/test/java/com/signicat/hystrix/servlet/MockHttpServletResponse.java]: This is noise based on your personal IDE prefs. Please revert this file.

File: src/test/java/com/signicat/hystrix/servlet/MockHttpServletRequest.java [hunk 3]
```diff
@@ -7,34 +7,15 @@
  */
 package com.signicat.hystrix.servlet;
 
+import javax.servlet.*;
+import javax.servlet.http.*;
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.UnsupportedEncodingException;
 import java.security.Principal;
-import java.util.Collection;
```
Comment (einarmr) [inline src/test/java/com/signicat/hystrix/servlet/MockHttpServletRequest.java]: This is noise based on your personal IDE prefs. Please revert this file.

File: src/test/java/com/signicat/hystrix/servlet/AsyncWrapperServletTest.java [hunk 4]
```diff
@@ -19,16 +19,15 @@
 import org.eclipse.jetty.servlet.ServletHolder;
 import org.junit.Test;
 
-import java.io.IOException;
-import java.util.concurrent.CountDownLatch;
```
Comment (einarmr) [inline src/test/java/com/signicat/hystrix/servlet/AsyncWrapperServletTest.java]: This is noise based on your personal IDE prefs. Please revert this file.

File: src/main/java/com/signicat/hystrix/servlet/TimeoutAwareHttpServletResponse.java [hunk 5]
```diff
@@ -7,15 +7,14 @@
  */
 package com.signicat.hystrix.servlet;
 
+import javax.servlet.ServletOutputStream;
```
Comment (einarmr) [inline src/main/java/com/signicat/hystrix/servlet/TimeoutAwareHttpServletResponse.java]: This is noise based on your personal IDE prefs. Please revert this file.

File: src/main/java/com/signicat/hystrix/servlet/TimeoutAwareHttpServletRequest.java [hunk 6]
```diff
@@ -7,6 +7,8 @@
  */
 package com.signicat.hystrix.servlet;
 
+import javax.servlet.*;
```
Comment (einarmr) [inline src/main/java/com/signicat/hystrix/servlet/TimeoutAwareHttpServletRequest.java]: This is noise based on your personal IDE prefs. Please revert this file.

File: src/main/java/com/signicat/hystrix/servlet/ConcurrencyStrategyWithCoreTimeOut.java [hunk 7]
```diff
@@ -16,44 +17,53 @@
...
+    private ConcurrencyStrategyWithCoreTimeOut() {
     }
 
-    public static ConcurrencyStrategyWithExplicitCoreSize getInstance() {
+    public static ConcurrencyStrategyWithCoreTimeOut getInstance() {
         return INSTANCE;
     }
 
+    // this might be worth keeping around if we ever decide to switch out HystrixThreadPoolDefault
+    // with an implementation that requires this overloaded variant
```
Comment (einarmr) [inline src/main/java/com/signicat/hystrix/servlet/ConcurrencyStrategyWithCoreTimeOut.java]: Keeping dead code around is never a good idea. If you don't use this, kill it. This is what revision control systems were made for.

File: src/main/java/com/signicat/hystrix/servlet/AsyncWrapperServlet.java [hunk 8]
```diff
@@ -7,34 +7,22 @@
  */
 package com.signicat.hystrix.servlet;
 
-import com.netflix.hystrix.Hystrix;
```
Comment (einarmr) [inline src/main/java/com/signicat/hystrix/servlet/AsyncWrapperServlet.java]: Clean the imports that need to be cleaned. The rest: Don't introduce unneccessary noise.

**Comment by magsel** [comment]
jeez sorry, jeg skal stenge denne. rotet dette til

**Comment by magsel** [comment]
du er skikkelig på! superkult

**Pull request closed without merging**